### PR TITLE
Add simple TabCounterButton to compose-tabstray.

### PR DESCRIPTION
**Pull request by pocmo**
🌮 🌮 🌮

### CODE REVIEW

**Review by jonalmeida — APPROVED** [review]
🔢

File: components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt [hunk 1]
```diff
@@ -63,7 +63,7 @@ class TabCounter @JvmOverloads constructor(
     @VisibleForTesting
     internal fun setColor(colorStateList: ColorStateList) {
         val tabCounterBox =
-            DrawableUtils.loadAndTintDrawable(context, R.drawable.mozac_ui_tabcounter_box, colorStateList)
+            DrawableUtils.loadAndTintDrawable(context, R.drawable.`mozac_ui_tabcounter_box`, colorStateList)
```
Comment (jonalmeida) [inline components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt:66]: Seems redundant?

**Comment by pocmo** [comment]
Needs ktlint update: #10675

File: components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt (hunk 1 above)
Comment (pocmo) [reply components/ui/tabcounter/src/main/java/mozilla/components/ui/tabcounter/TabCounter.kt:66]: 🤔 Not sure where that came from.

**Pull request merged**